### DISCUSSION TITLE
docker paid any alternatives

### DISCUSSION THREAD

**Discussion by riteshmk** [amundsen-io/amundsen#1565] [Q&A]
Hi,

i wish to pick and use this for enterprise purpose. The current version is purely dependent on docker. Is there anyone who can help me if i can install this as standalone. I'm okay to install individual components.

i plan to use it for production scenario

Appreciate your support and guidance

Cheers!!

**Comment by tenzo** [comment]
I suggest using Kubernetes for running Amundsen - no licensing issues while using it.
For local development (if using e.g. MacOS) I use Rancher Desktop which comes with Kubernetes (k3s) virtual machine and docker / containerd (nerdctl) tools.

The Helm charts for Kubernetes are here: https://github.com/amundsen-io/amundsen/tree/main/amundsen-kube-helm
